pkgs/applications/editors/neovim/nightly.nix

```diff
@@ -0,0 +1,143 @@
+{ lib, stdenv, fetchFromGitHub, cmake, gettext, msgpack, libtermkey, libiconv
```

Comment:
Except for the `tree-sitter` portions this is essentially copy-paste of `default.nix`